### DISCUSSION TITLE
How to become a module contributor?

### DISCUSSION THREAD

**Discussion by msl0** [Azure/Azure-Verified-Modules#1766] [Q&A]
Hello!
I like the AVM project, but I encounter various minor bugs in many modules. I can solve most of them, but communication is a challenge. I encounter modules with open PRs that have been around for days/months and have not been reviewed - no feedback from module owners. I know that as a non-Microsoft FTE, I cannot become a co-owner of the module, but what do I need to do to become a module contributor so that I can review PRs and approve Github Action workflows to help the AVM project grow?

**Comment by mloskot** [comment]
I guess, helping out with orphaned modules and earning reputation to become an owner seems like an open source way, see https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/avm-issue-triage/#orphaned-modules

Disclaimer: AVM observer here, not contributor, yet.

**Reply by msl0** [reply]
I've learned in the meantime, Microsoft's policy currently does not allow granting approval rights for a non-Microsoft FTE. It is possible to tag someone else from the AVM team if the owner is not available for a significantly longer time
